### ISSUE TITLE
Add accessibility semantics to gesture detector widgets

### ISSUE THREAD

**Issue by timukasr** [daohoangson/flutter_widget_from_html#1426]
## Use case

When HTML contains links, then they should have button (or link) semantics so screen readers announce them correctly and Voice Control has item labels. This can currently be achieved via custom `factoryBuilder`.

## Proposal

I think this should be a standard behavior: wrapping `MouseRegion` with `Semantics(button: true)` or similar.

https://github.com/daohoangson/flutter_widget_from_html/blob/a564ad829bb26cf2360306e1a2318081d9d07225/packages/core/lib/src/core_widget_factory.dart#L261-L276
